src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -1,44 +1,36 @@
 package racingcar.domain
 
-import racingcar.view.OutputView
-
 class Cars(nameOfCars: List<String>, movingConditionStrategy: MovingConditionStrategy) {
     private val cars = mutableListOf<Car>()
+    private var movingConditionStrategy: MovingConditionStrategy
```

Comment:
```suggestion
    private val movingConditionStrategy: MovingConditionStrategy
```
변경 불가능한 프로퍼티로 선언해도 될것 같아요 😄